frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -173,14 +173,45 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
     );
 
     return (
-      <Collapse defaultActiveKey={["1", "2", "3", "4"]}>
+      <Collapse bordered={false} defaultActiveKey={["1", "2", "3", "4"]}>
         <Panel header="Color Layers" key="1">
           {colorSettings}
         </Panel>
         {this.props.hasSegmentation ? this.getSegmentationPanel() : null}
-        <Panel header="Quality" key="3">
+        <Panel header="Data Rendering" key="3">
+          <DropdownSetting
+            label={settings.quality}
+            value={this.props.datasetConfiguration.quality}
+            onChange={_.partial(this.onChangeQuality, "quality")}
+          >
+            <Option value="0">High</Option>
+            <Option value="1">Medium</Option>
+            <Option value="2">Low</Option>
+          </DropdownSetting>
+          <DropdownSetting
+            label={
+              <React.Fragment>
+                {settings.loadingStrategy}{" "}
+                <Tooltip title={settings.loadingStrategyDescription}>
+                  <Icon type="info-circle" />
+                </Tooltip>
+              </React.Fragment>
+            }
+            value={this.props.datasetConfiguration.loadingStrategy}
+            onChange={_.partial(this.props.onChange, "loadingStrategy")}
+          >
+            <Option value="BEST_QUALITY_FIRST">Best quality first</Option>
+            <Option value="PROGRESSIVE_QUALITY">Progressive quality</Option>
+          </DropdownSetting>
           <SwitchSetting
-            label={settings.fourBit}
+            label={
+              <React.Fragment>
+                {settings.fourBit}{" "}
+                <Tooltip title="Load each data bucket in half quality. Recommended for poor and/or capped Internet connections.">
```

Comment:
Maybe "Decrease data bucket size by half using lossy compression. Recommended ...", but just a suggestion :)

or "Halve the size of data buckets using (lossy) compression. Recommended ..."